release/models/platform/openconfig-platform-storage.yang

```diff
@@ -0,0 +1,75 @@
+module openconfig-platform-storage {
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://openconfig.net/yang/platform/storage";
+
+  prefix "oc-storage";
+
+  import openconfig-platform {
+    prefix oc-platform;
+  }
+  import openconfig-extensions {
+    prefix oc-ext;
+  }
+  import openconfig-yang-types {
+    prefix oc-yang;
+  }
+
+  // meta
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+     www.openconfig.net";
+
+  description
+    "This module defines data related to STORAGE components in the
+     OpenConfig platform model.";
+
+  oc-ext:openconfig-version "0.1.0";
+
+  revision "2024-04-24" {
+    description
+      "Initial revision.";
+    reference
+      "0.1.0";
+  }
+
+  // OpenConfig specific extensions for module metadata.
+  oc-ext:regexp-posix;
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  // identity statements
+  // typedef statements
+  // grouping statements
+  grouping storage-state {
+    description
+      "Storage component state.  These counters are derived from the
+       linux kernel block layer statistics in /sys/block/<dev>/stat.";
+    reference
+      "https://www.kernel.org/doc/Documentation/block/stat.txt";
+
+    leaf io-errors {
```

Comment:
This augments directly into `/components/component/storage/state` but it is a counter value.  The design of other OC modeling would nest this under a `counters` container

Reply:
Thanks for this.  I also prefer using the counters container as most of OC does this.